src/Data/Nat/Divisibility/Core.agda

```diff
@@ -35,9 +38,8 @@ record _∣_ (m n : ℕ) : Set where
   constructor divides
   field quotient : ℕ
         equality : n ≡ quotient * m
-open _∣_ using (quotient) public
```

Comment:
Ah so why have you removed the public export here? This is not a full-blown breaking change as we don't officially encourage the use of `Core` modules, but the nonetheless it's big enough that we should have a good reason for doing so!

Reply:
Two reasons:
* It now gets publicly exported from `Divisibility` instead (and so, in theory, should be 'better' than relying on a `Core` module)
* If it's exported here, then the lemmas above in the anonymous module as phrased end up with ambiguous resolution of the projection name `quotient`... and thus was my 'solution' to that...

Happy to take a steer on how best to handle this, but some of these things arise, I think,  precisely from your earlier suggestion to take all of this out of `Core`, where I had previously had put the #1711 lemmas as manifest fields of the record...

Reply:
Reverted by commit 9708ca8a6